tmail-backend/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/tmail/james/common/LinagoraForwardGetMethodContract.scala

```diff
@@ -0,0 +1,855 @@
+
+import com.linagora.tmail.james.common.LinagoraForwardGetMethodContract.{basePath, webAdminApi}
+import io.netty.handler.codec.http.HttpHeaderNames.ACCEPT
+import io.restassured.RestAssured.{`given`, requestSpecification}
+import io.restassured.http.ContentType.JSON
+import io.restassured.specification.RequestSpecification
+import net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson
+import org.apache.http.HttpStatus.{SC_NO_CONTENT, SC_OK}
+import org.apache.james.GuiceJamesServer
+import org.apache.james.jmap.core.ResponseObject.SESSION_STATE
+import org.apache.james.jmap.core.UuidState.INSTANCE
+import org.apache.james.jmap.http.UserCredential
+import org.apache.james.jmap.rfc8621.contract.Fixture.{ACCEPT_RFC8621_VERSION_HEADER, ANDRE, BOB, BOB_PASSWORD, CEDRIC, DOMAIN, authScheme, baseRequestSpecBuilder}
+import org.apache.james.utils.{DataProbeImpl, WebAdminGuiceProbe}
+import org.apache.james.webadmin.WebAdminUtils
+import org.junit.jupiter.api.{BeforeEach, Test}
+
+object LinagoraForwardGetMethodContract {
+  private var webAdminApi: RequestSpecification = _
+  private val basePath: String = s"/address/forwards/${BOB.asString}/targets"
+}
+
+trait LinagoraForwardGetMethodContract {
+  @BeforeEach
+  def setUp(server : GuiceJamesServer): Unit = {
+    server.getProbe(classOf[DataProbeImpl])
+      .fluent()
+      .addDomain(DOMAIN.asString)
+      .addUser(BOB.asString(), BOB_PASSWORD)
+
+    requestSpecification = baseRequestSpecBuilder(server)
+      .setAuth(authScheme(UserCredential(BOB, BOB_PASSWORD)))
+      .build()
+
+    webAdminApi = WebAdminUtils.buildRequestSpecification(server.getProbe(classOf[WebAdminGuiceProbe]).getWebAdminPort)
+      .setBasePath(basePath)
+      .build()
+  }
+
+  @Test
+  def forwardGetShouldReturnEmptyListByDefault(): Unit = {
+    val response = `given`
+      .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .body(s"""{
+               |  "using": [
+               |    "urn:ietf:params:jmap:core",
+               |    "com:linagora:params:jmap:forward"],
+               |  "methodCalls": [[
+               |    "Forward/get",
+               |    {
+               |      "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+               |      "ids": null
+               |    },
+               |    "c1"]]
+               |}""".stripMargin)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .contentType(JSON)
+      .extract
+      .body
+      .asString
+
+    assertThatJson(response).isEqualTo(
+      s"""{
+         |  "sessionState": "${SESSION_STATE.value}",
+         |  "methodResponses": [[
+         |    "Forward/get",
+         |    {
+         |      "accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+         |      "state": "${INSTANCE.value}",
+         |      "list": [
+         |        {
+         |          "id":"singleton",
+         |          "localCopy": false,
```

Comment:
IMO by default when there's no forwars we should say "local copy" -> true as the guy will receive the mail.

Reply:
The local copy radio button seemed completely independent of that on esn, thus I kept the behavior. But your remark makes sense and I don't mind. Will check